cmake/SelectPcsx2Plugins.cmake

```diff
@@ -71,8 +71,8 @@ else()
     set(pcsx2_core FALSE)
     print_dep("Skip build of pcsx2 core: miss some dependencies" "${msg_dep_pcsx2}")
 endif()
-# Linux need also gtk2
-if(UNIX AND pcsx2_core AND NOT GTKn_FOUND)
+# Linux, BSD, use gtk2, but not OSX
+if(UNIX AND pcsx2_core AND NOT GTKn_FOUND AND NOT APPLE)
```

Comment:
Instead to duplicate all if entries. I propose to set GTKn_FOUND to true when APPLE. Just add a comment.